bool link
int32 mod
int32 moto_1_1
int32 moto_1_2
int32 moto_1_3
int32 moto_1_4
float64 turn
